avocado/plugins/runners/exec_test.py

```diff
@@ -122,6 +122,9 @@ def _get_env(self, runnable):
         env = dict(os.environ)
         if runnable.kwargs:
             env.update(runnable.kwargs)
+            for key, value in runnable.kwargs.items():
+                if value is None:
```

Comment:
I'd be more explicit here and write this as:

```python
for key, value in runnable.kwargs.items():
   if value is None and key in env:
      del(env[key])
```

The reason being there's no need to return any value (which `pop()` does) and the `None` value as a safeguard makes reading the goal of the `pop()` confusing.